analyzer.go

```diff
@@ -0,0 +1,206 @@
+package wraperr
+
+import (
+	"bufio"
+	"go/ast"
+	"go/token"
+	"go/types"
+	"os"
+	"strings"
+
+	"golang.org/x/tools/go/analysis"
+	"golang.org/x/tools/go/analysis/passes/inspect"
+	"golang.org/x/tools/go/ast/inspector"
+)
+
+var Analyzer = &analysis.Analyzer{
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var Analyzer should have comment or be unexported